src/DocBlock/Tags/InvalidTag.php

```diff
@@ -104,7 +104,9 @@ static function (&$value) : void {
         do {
             $trace = array_map(
                 static function (array $call) use ($flatten) : array {
-                    array_walk_recursive($call['args'], $flatten);
+		    $args = $call['args'] ?? [];
```

Comment:
Please use 4 spaces for indentation, and not tabs.

Reply:
It was fixed a couple of seconds after committing, sorry.